api/v1alpha1/kafkatopic_types.go

```diff
@@ -41,6 +41,10 @@ type KafkaTopicSpec struct {
 // +k8s:openapi-gen=true
 type KafkaTopicStatus struct {
 	State TopicState `json:"state"`
+	// ManagedBy describes who is the manager of the Kafka topic.
+	// When its value is not 'koperator' then modifications to the KafkaTopic CR's topic configurations will be unaffected on the Kafka topic.
```

Comment:
Sorry, I'm a bit confused by this still. What happens if the annotation is not set at all? Do we assume it's managed by koperator or not? I seem to remember a comment (though I can't find it) that said that previously.

Reply:
Yes, when there is no annotation it is managedby koperator